pkg/flink/types.go

```diff
@@ -63,6 +63,15 @@ func GetDefaultLabels(taskCtx pluginsCore.TaskExecutionMetadata) Labels {
 	)
 }
 
+type EnvironmentVariables map[string]string
+
+func GetDefaultEnvironmentVariables(taskCtx pluginsCore.TaskExecutionMetadata) EnvironmentVariables {
+	return utils.UnionMaps(
+		config.GetK8sPluginConfig().DefaultEnvVars,
+		utils.CopyMap(taskCtx.GetEnvironmentVariables()),
+	)
+}
+
```

Comment:
Hi @andresgomezfrr is this still relevant? can you expand on why is this needed?